dataset/src/test/scala/frameless/DropTest.scala

```diff
@@ -0,0 +1,24 @@
+package frameless
+
+import org.scalacheck.Prop
+import org.scalacheck.Prop._
+
+class DropTest extends TypedDatasetSuite {
+  test("drop five columns") {
```

Comment:
Maybe you could add a test dropping the only the first column, and another one dropping only the last column

Reply:
Sure, I can make it more explicit by making separate tests but those two cases are already covered in this test by dropping `'a` which is the first column and then `_4` etc which are the last ones